mlrun/platforms/iguazio.py

```diff
@@ -424,6 +423,61 @@ def dump_record(rec):
             )
 
 
+class KafkaOutputStream:
+    def __init__(
+        self,
+        topic,
+        brokers,
+        producer_options=None,
+        mock=False,
+    ):
+        self._kafka_producer = None
+        self._topic = topic
+        self._brokers = brokers
+        self._producer_options = producer_options
+
+        self._mock = mock
+        self._mock_queue = []
+
+        self._initialized = False
+
+    def _lazy_init(self):
+        if self._initialized:
+            return
+
+        import kafka
+
+        self._kafka_producer = kafka.KafkaProducer(
+            bootstrap_servers=self._brokers,
+            **self._producer_options,
```

Comment:
`self._producer_options` may be None at this point, which will cause this code to explode. See suggestion above.